src/apps/api/views/mapping.py

```diff
@@ -1,83 +1,128 @@
+from common.dal.sample_da import Sample
 from common.schema_versions.lookup.dtol_lookups import DTOL_EXPORT_TO_STS_FIELDS, STANDARDS, STANDARDS_MAPPING_FILE_PATH
 from django.http import HttpResponse
-from src.apps.api.utils import finish_request
 
 import bson.json_util as jsonb
-import common.schemas.utils.data_utils as d_utils
 import json
 
-def get_standard_data(standard_list=STANDARDS, manifest_type=str(), queryByManifestType=False):
-    with open(STANDARDS_MAPPING_FILE_PATH) as f:
-        standards_file_data = json.load(f)
-        standards_file_data = standards_file_data[0]
-        output = list()
-        output_dict = dict()
-
-        if standard_list and not queryByManifestType:
-            for tol_column, standard_data in standards_file_data.items():
-                tol_column_dict = dict()
-                tol_column_dict[tol_column] = dict()
-                
-                for standard in standard_list:
-                    if standard_data.get(standard, str()):
-                        tol_column_dict[tol_column] |= {standard: {'key': standard_data.get(standard,str()).get('field', str())}}
-                            
-                        # Merge dictionaries
-                        output_dict |= tol_column_dict
-                            
-            output.append(output_dict)
-
-            result = finish_request(template=output, error=None, num_found=len(output[0]), return_http_response=False)
+def get_standard_fields(standard, standards_data):
+    fields = list()
+    
+    for key in standards_data:
+        # Retrieve the field for the standard if it exists
+        field = standards_data[key].get(standard, {}).get('field', '')
+        
+        if field:
+            fields.append(field)
 
-        elif any(x in standard_list for x in STANDARDS) and queryByManifestType and manifest_type:
-            manifest_type_columns = DTOL_EXPORT_TO_STS_FIELDS.get(manifest_type, str())
+    return fields
 
-            for tol_column, standard_data in standards_file_data.items():
-                tol_column_dict = dict()
-                tol_column_dict[tol_column] = dict()
+def get_mapped_fields_for_project(standard, project, is_query_by_project):
```

Comment:
try to eliminate  the function parameters if they depend on each other.  For example, "is_query_by_project"  depends on project.  if "project", "is_query_by_project" is true.   In this case, you just need to check "project" only.

If you have more parameters, you will have cases to handle.   

A lazy programer is to use less code to do the same thing, such that the code is cleaner and more readable.